src/ray/raylet/worker_pool.cc

```diff
@@ -198,14 +198,13 @@ void WorkerPool::update_worker_startup_token_counter() {
 
 void WorkerPool::AddWorkerProcess(
     State &state,
-    const int workers_to_start,
+    const bool is_pending_registration,
```

Comment:
Why do we need this new `is_pending_registration` parameter? Shouldn't it always be true?

Reply:
Removed.